docs/common/releases/release-notes.md

```diff
@@ -66,7 +66,21 @@ To add a dependency on Iceberg in Maven, add the following to your `pom.xml`:
 </dependencies>
 ```
 
-## 0.13.0 Release Notes
+## 0.13.1 Release Notes
+
+**Important bug fixes:**
+
+* **Spark**
+  * Fix predicate pushdown (filter-based and merge conditions) for row-level operations in Spark 3.2. [[\#4023](https://github.com/apache/iceberg/pull/4023)]
+  * Fix an issue introduced in [#3722](https://github.com/apache/iceberg/pull/3722) and in the 0.13.0 release which lead to not being able to
+  create a table in the root namespace of the Hadoop catalog.[[\#3986](https://github.com/apache/iceberg/pull/3986)]
+
+* **Flink**
+  * Fix manifest location collision when there are multiple committers for multiple sink tables. [[\#4024](https://github.com/apache/iceberg/pull/4024)]
+
+## Past releases
+
+### 0.13.0 Release Notes
 
 Apache Iceberg 0.13.0 was released on February 4th, 2022.
```

Comment:
need to add artifact links for past releases

Reply:
Good point, added.